libs/wire-subsystems/test/unit/Wire/UserSubsystem/InterpreterSpec.hs

```diff
@@ -365,8 +365,8 @@ spec = describe "UserSubsystem.Interpreter" do
     describe "Scim+UpdateProfileUpdate" do
       prop
         "Updating handles fails when UpdateOriginWireClient"
-        \(alice, Handle newHandle, domain, config) ->
-          not (isBlacklistedHandle (Handle newHandle)) ==>
+        \(alice, fromHandle -> newHandle, domain, config) ->
+          not (isBlacklistedHandle (fromJust (parseHandle newHandle))) ==>
```

Comment:
Would it make sense to change `isBlacklistedHandle` to take a `Handle` instead of parsing it again here?